Makefile

```
@@ -1,50 +1,76 @@
+	@echo "  make test-coverage           - Run all tests with coverage report"
+	@echo "  make test-coverage-html      - Run all tests and generate HTML coverage report"
+	@echo "  make test-clean              - Clean test cache and run tests"
+	@echo "  make mock                    - Generate mock implementations"
+	@echo "  make install-pre-push-hook   - Install the pre-push git hook"
+	@echo "  make uninstall-pre-push-hook - Uninstall the pre-push git hook"
 
 lint:
 	golangci-lint run ./...
 
-test-all:
+test:
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (6).

(maxbodylength)